src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,342 @@
+var malawi_binary = ee.Image("projects/ee-aasareansah/assets/malawi_binary"),
+    geometry = /* color: #98ff00 */ee.Geometry.MultiPoint(
```

Comment:
These multipoints don't need to be in this script as they are specific to Malawi